corporate/lib/support.py

```diff
@@ -135,6 +145,11 @@ def get_current_plan_data_for_support_view(billing_session: BillingSession) -> P
 
 
 def get_data_for_support_view(billing_session: BillingSession) -> SupportData:
+    if isinstance(billing_session, RemoteServerBillingSession):
+        user_data = get_remote_server_guest_and_non_guest_count(billing_session.remote_server.id)
+    else:  # nocoverage
+        assert isinstance(billing_session, RemoteRealmBillingSession)
+        user_data = get_remote_realm_guest_and_non_guest_count(billing_session.remote_realm)
```

Comment:
It would be nice to also show if they are not current on uploading the audit log data powering this.